extra/modules/confiant-ad-quality/src/main/java/org/prebid/server/hooks/modules/com/confiant/adquality/core/RedisVerticle.java

```diff
@@ -0,0 +1,103 @@
+package org.prebid.server.hooks.modules.com.confiant.adquality.core;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.redis.client.Redis;
+import io.vertx.redis.client.RedisAPI;
+import io.vertx.redis.client.RedisConnection;
+import io.vertx.redis.client.RedisOptions;
+import org.prebid.server.hooks.modules.com.confiant.adquality.model.RedisRetryConfig;
+
+public class RedisVerticle extends AbstractVerticle {
```

Comment:
Why do you need a separate verticle just to start redis on event loop? Consider looking into pbs `ContextRunner` class usages. 
Also, this class should be just a part of redis client class.